internal/api/describe_test.go

```diff
@@ -14,23 +14,36 @@ func TestDescribeFull(test *testing.T) {
 		test.Fatalf("Unable to get the API description filepath: '%v'.", err)
 	}
 
-	var expectedDescription core.APIDescription
-	err = util.JSONFromFile(path, &expectedDescription)
+	var expectedDescriptions core.APIDescription
+	err = util.JSONFromFile(path, &expectedDescriptions)
 	if err != nil {
 		test.Fatalf("Failed to load api.json: '%v'.", err)
 	}
 
-	fullDescription := Describe(*GetRoutes())
+	descriptions, err := Describe(*GetRoutes())
+	if err != nil {
+		test.Fatalf("Failed to describe endpoints: '%v'.", err)
+	}
+
+	for endpoint, description := range descriptions.Endpoints {
+		if description.Description == "" {
+			test.Errorf("Describe found an empty description. Endpoint: '%s'.", endpoint)
+			continue
+		}
+	}
```

Comment:
I think this should be in another test (as opposed to being grouped in this other test).
Maybe "TestDescribeEmpty"?